lib/msal-common/src/authority/Authority.ts

```diff
@@ -1184,9 +1234,7 @@ export class Authority {
      * @param authority
      */
     static transformCIAMAuthority(authority: string): string {
-        let ciamAuthority = authority.endsWith(Constants.FORWARD_SLASH)
```

Comment:
Adding forward slash moved to `formatAuthorityUri` so it can be reused and called before passing authority into this method